tasks/src/video-classification/data.ts

```diff
@@ -66,7 +66,16 @@ const taskData: TaskDataCustom = {
 			id:          "microsoft/xclip-base-patch32",
 		},
 	],
-	spaces:       [],
+	spaces:       [
+		{
+			description: "An application that classifies video at different timestamps.",
+			id:          "nateraw/lavila",
+		},
+		{
+			description: "An application that classifies video.",
+			id:          "fcakyon/video-classification",
```

Comment:
Just ideating.

We also have a zero-shot video classification demo here: https://huggingface.co/spaces/fcakyon/zero-shot-video-classification. Pretty cool!

We don't have a task for it yet since we don't have many models for the task yet. Would be good revisit it :)